integration-tests/bats/sql-remotes-server.bats

```diff
@@ -0,0 +1,556 @@
+#!/usr/bin/env bats
+load $BATS_TEST_DIRNAME/helper/common.bash
+
+remotesrv_pid=
+setup() {
+    setup_common
+    cd $BATS_TMPDIR
+    mkdir remotes-$$
+    mkdir remotes-$$/empty
+    echo remotesrv log available here $BATS_TMPDIR/remotes-$$/remotesrv.log
+    remotesrv --http-port 1234 --dir ./remotes-$$ &> ./remotes-$$/remotesrv.log 3>&- &
+    remotesrv_pid=$!
+    cd dolt-repo-$$
+    mkdir "dolt-repo-clones"
+}
+
+teardown() {
+    teardown_common
+    kill $remotesrv_pid
+    rm -rf $BATS_TMPDIR/remotes-$$
+}
+
+@test "sql-remotes-server: push with only one argument" {
+    skip "todo implement SQL push, pull, fetch"
+    dolt remote add test-remote http://localhost:50051/test-org/test-repo
+    run dolt sql -q "select dolt_push('test-remote')"
+    [ "$status" -eq 1 ]
+}
+
+@test "sql-remotes-server: push and pull master branch from a remote" {
+    dolt remote add test-remote http://localhost:50051/test-org/test-repo
+    # TODO dolt_push
+    #run dolt sql -q "select dolt_push('test-remote', 'master')"
+    run dolt push test-remote master
+    [ "$status" -eq 0 ]
+    [ -d "$BATS_TMPDIR/remotes-$$/test-org/test-repo" ]
+    run dolt sql -q "select dolt_pull('test-remote')"
+    [ "$status" -eq 0 ]
+    #[[ "$output" =~ "Everything up-to-date" ]] || false
+}
+
+@test "sql-remotes-server: push and pull non-master branch from remote" {
+    skip "todo implement SQL push, pull, fetch"
```

Comment:
The skip messages on these are too general, they all say the same thing.

You've already implemented pull, e.g. and not all the tests need both push and fetch